server/graphql/v2/interface/Account.js

```diff
@@ -394,36 +393,37 @@ const accountFieldsDefinition = () => ({
         );
       }
 
-      const collectives = await sequelize.query(
-        `
-        SELECT
-          c.*, COUNT(*) OVER() AS __total_count__
-        FROM
-          "Collectives" c
-        INNER JOIN "Expenses" AS "submittedExpenses"
-          ON c."id" = "submittedExpenses"."FromCollectiveId"
-          AND "submittedExpenses"."deletedAt" IS NULL
-        INNER JOIN "VirtualCards" AS vc
-          ON "submittedExpenses"."VirtualCardId" = vc."id"
-          AND vc."deletedAt" IS NULL
-          AND (vc."data"#>>'{type}') = 'MERCHANT_LOCKED'
-        WHERE c."deletedAt" IS NULL
-          AND c."type" = 'VENDOR'
-          AND "submittedExpenses"."CollectiveId" = :collectiveId
-        GROUP BY c.id
-        LIMIT :limit
-        OFFSET :offset
-      `,
-        {
-          type: sequelize.QueryTypes.SELECT,
-          model: models.Collective,
-          mapToModel: true,
-          replacements: { collectiveId: account.id, limit: args.limit, offset: args.offset },
+      const result = await models.Collective.findAndCountAll({
+        group: 'Collective.id',
+        where: {
+          type: CollectiveTypes.VENDOR,
         },
-      );
+        include: [
+          {
+            attributes: [],
+            association: 'submittedExpenses',
+            required: true,
+            include: [
+              {
+                attributes: [],
+                association: 'virtualCard',
+                required: true,
+                where: {
+                  CollectiveId: account.id,
+                  data: { type: 'MERCHANT_LOCKED' },
+                },
+              },
+            ],
+          },
+        ],
+      });
 
-      const totalCount = get(collectives, '0.dataValues.__total_count__', 0);
-      return { nodes: collectives, totalCount, limit: args.limit, offset: args.offset };
+      return {
+        nodes: result.rows,
+        totalCount: result.count.length, // See https://github.com/sequelize/sequelize/issues/9109
```

Comment:
@SudharakaP Because of [this bug](https://github.com/sequelize/sequelize/issues/9109) in sequelize when using `GROUP BY` with `findAndCountAll`, we have to hack things a little to make it work.